### PR TITLE
Updated License

### PR DESCRIPTION
**Pull request by VincentRPS**
## Summary

Chore: Update Licenses To All Files

## Checklist

- [x] This PR Updates license info

### CODE REVIEW

File: nextcord/__init__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 A basic wrapper for the Discord API.
 
-:copyright: (c) 2015-present Rapptz
+:copyright: (c) 2015-2021 Rapptz & 2021-present tag-epic
```
Comment (veni-vidi-code) [inline nextcord/__init__.py:7]: and from 2021 on?

**Review by veni-vidi-code — CHANGES_REQUESTED** [review]
I would swap the order (first epic then Rapptz

File: nextcord/__init__.py (hunk 1 above)
Comment (VincentRPS) [reply nextcord/__init__.py:7]: Well only this one cause i am not spending another 20 minutes copy-pasting license info again

**Comment by TAG-Epic** [comment]
Let's only add my copyright to files I've modified.

File: nextcord/__init__.py (hunk 1 above)
Comment (veni-vidi-code) [reply nextcord/__init__.py:7]: this one here is different from the others and should definitly be changed

Comment (VincentRPS) [reply nextcord/__init__.py:7]: yeah

> this one here is different from the others and should definitly be changed

**Pull request closed without merging**